cmd/aws-sso-creds/main.go

```diff
@@ -19,6 +19,7 @@ var (
 	ssoURL                  = getopt.StringLong("SSOUrl", 'u', "", "The SSO URL(https://<my-domain>.awsapps.com/start) the AWS_SSO_START_URL env var has precedence over this flag")
 	ssoRegion               = getopt.StringLong("SSORegion", 'r', "", "The AWS SSO region, the AWS_SSO_REGION env var has precedence over this flag")
 	forceLogin              = getopt.BoolLong("forceAuth", 'f', "Force Authentication with AWS SSO")
+	noInteractive           = getopt.BoolLong("noInteractive", 0, "Do not open in the browser automatically")
```

Comment:
Replace no with non in noInteractive (i.e. nonInteractive) or even replace it with noBrowser (Duplicate)

Reply:
Done I made the change